site/src/components/roadmap/style.module.css

```diff
@@ -0,0 +1,79 @@
+.heading {
+  margin-bottom: 5%;
+  text-align: center;
+}
+
+.heading3 {
+  padding-bottom: 5%;
+}
+
+.spacing {
+  border-radius: 8px;
+}
+
+.cardContainer:global(.saltGridLayout) {
+  margin-top: 50px;
+  display: flex;
+  flex-wrap: wrap; /* Allow items to wrap to the next line */
+  gap: 20px; /* Adjust the gap between columns */
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.content {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+}
+
+.bar {
+  flex: 1;
+  max-width: 400px;
+  width: 100%;
+}
+
+.test {
+  width: "400px";
+  margin-bottom: " 50px ";
+}
+.date {
+  display: inline;
+}
+
+.icon {
+  margin-bottom: -3px;
+  margin-left: 9px;
+}
+.column {
+  padding-right: 20px; /* Adjust the padding as per your design */
+}
+
+.info {
+  margin-left: 20px;
+  background-color: rgb(200, 205, 212);
```

Comment:
this component should work in light and dark mode... I would look at any magic numbers in this file and see if there is anything which can be replaced by theme or made more responsive